src/modules/launcher/Wox.Plugin/Common/Win32/NativeMethods.cs

```diff
@@ -493,29 +493,29 @@ public enum DwmNCRenderingPolicies
     public enum DwmWindowAttributes
     {
         NCRenderingEnabled = 1,
-        NCRenderingPolicy,
-        TransitionsForceDisabled,
-        AllowNCPaint,
-        CaptionButtonBounds,
-        NonClientRtlLayout,
-        ForceIconicRepresentation,
-        Flip3DPolicy,
-        ExtendedFrameBounds,
-        HasIconicBitmap,
-        DisallowPeek,
-        ExcludedFromPeek,
-        Cloak,
-        Cloaked,
-        FreezeRepresentation,
-        PassiveUpdateMode,
-        UseHostbackdropbrush,
-        UseImmersiveDarkMode,
-        WindowCornerPreference,
-        BorderColor,
-        CaptionColor,
-        TextColor,
+        NCRenderingPolicy = 2,
+        TransitionsForceDisabled = 3,
+        AllowNCPaint = 4,
+        CaptionButtonBounds = 5,
+        NonClientRtlLayout = 6,
+        ForceIconicRepresentation = 7,
+        Flip3DPolicy = 8,
+        ExtendedFrameBounds = 9,
+        HasIconicBitmap = 10,
+        DisallowPeek = 11,
+        ExcludedFromPeek = 12,
+        Cloak = 13,
+        Cloaked = 14,
```

Comment:
Do we use a different enum in ww plugin? otherwise please verify that dummy windows for uwp apps shown in results.

Reply:
ww plugin just puts 12 directly 😆

Reply:
I was trying to use CaptionColor and that's when I saw this was wrong.

Reply:
> ww plugin just puts 12 directly

No. See `Components\Window.cs:L306`

Reply:
I know :) Cloak and Cloaked get the same value due to the default +1 logic in enums. This just makes it explicit since not every value follows that logic

Reply:
Thought that the PR moves the counter +2 as the first enum item now starts with `=2` instead of `=0` (no explicit definition).